docs/sources/reference/api/docker_remote_api.md

```diff
@@ -46,6 +46,22 @@ You can still call an old version of the API using
 
 ### What's new
 
+`GET /containers/(id)/copy`
+
+`PUT /containers/(id)/copy`
```

Comment:
The `POST /containers/(id)/copy` endpoint is still around for backwards compatibility, I don't want to change the behavior of that endpoint. Is there any reason to not create `PUT` endpoints other than the fact that there aren't any others yet?